addons/mail/static/src/models/call_view.js

```diff
@@ -128,44 +127,22 @@ registerModel({
         },
         /**
          * @private
+         * @returns {FieldCommand}
          */
         _computeTileParticipantCards() {
             if (!this.threadView) {
                 return clear();
             }
-            if (this.activeRtcSession && !this.hasSidebar) {
+            if (this.activeChannelParticipant && !this.hasSidebar) {
                 return clear();
             }
             const tileCards = [];
-            const sessionPartners = new Set();
-            const sessionGuests = new Set();
-            for (const rtcSession of this.channel.rtcSessions) {
-                if (this.filterVideoGrid && !rtcSession.videoStream) {
+            for (const channelCallParticipant of this.channel.channelCallParticipants) {
+                if (this.filterVideoGrid && !channelCallParticipant.callParticipant.isStreaming) {
```

Comment:
`isStreaming` should be in relation to a channel, so should be moved to `channelCallParticipant` instead
OR
callParticipant should include the notion of channel, which is only possible if it is using ChannelMember (channel.partner) instead of persona directly.